### ISSUE TITLE
Create installable package for administrative tools

### ISSUE THREAD

**Issue by celskeggs** [sipb/homeworld#71]
Once #52 is complete, it will be important that cluster administrators can easily install the administrative tools necessary for controlling the cluster. We should have a debian package containing:

* the relevant scripts for #52
* all deployment scripts and tools for homeworld in general
* dependencies on particular packages that provide necessary upstream binaries for administering the cluster

(This has been split out of #53, and depends on #52.)

EDIT: A rule of thumb: a cluster should be able to be deployed without having a cloned repository.

**Comment by celskeggs** [comment]
Deferred to DC5.

**Comment by celskeggs** [comment]
Done as part of the `spire` command transition in DC5. See the `feature-admin-tools` branch.
